src/Symfony/Component/AssetMapper/ImportMap/ImportMapManager.php

```diff
@@ -457,4 +429,13 @@ private function findAsset(string $path): ?MappedAsset
 
         return $this->assetMapper->getAssetFromSourcePath($path);
     }
+
+    private function createMissingImportMapAssetException(ImportMapEntry $entry): \InvalidArgumentException
+    {
+        if ($entry->isRemotePackage()) {
+            throw new \InvalidArgumentException(sprintf('The "%s" vendor asset is missing. Run "php bin/console importmap:install".', $entry->importName));
```

Comment:
Trying to decouple from `php bin/console`, since it could very well be `symfony console` instead:
```suggestion
            throw new \LogicException(sprintf('The "%s" vendor asset is missing. Try running the "importmap:install" command.', $entry->importName));
```